Source/WebCore/css/PropertySetCSSStyleDeclaration.h

```diff
@@ -53,7 +53,7 @@ class PropertySetCSSStyleDeclaration : public CSSStyleDeclaration {
     StyleSheetContents* contextStyleSheet() const;
 
 protected:
-    enum MutationType { NoChanges, PropertyChanged };
+    enum class MutationType : uint8_t { NoChanges, DirtyStyleAttribute, PropertyChanged };
```

Comment:
DirtyStyleAttribute -> StyleAttributeChanged would be more inline with the rest.

Reply:
or something like `StyleAttributeChangedWithoutChangingProperty` if we want to be verbose